Packs/Base/Scripts/DBotTrainTextClassifierV2/DBotTrainTextClassifierV2.yml

```diff
@@ -182,7 +182,7 @@ tags:
 timeout: 12µs
 type: python
 subtype: python2
-dockerimage: demisto/dl:1.4
+dockerimage: demisto/dl:languages1.0
```

Comment:
@erezh31 @lperryp is this ok ?

Reply:
Actually dl:languages1.0 is much heavier than 1.4, it's better to leave it as it @Itay4 @jochman